utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryClient.scala

```diff
@@ -4,6 +4,8 @@ import cats.data.Validated
 
 trait SchemaRegistryClient extends Serializable {
 
+  def getSchemaById(id: Int): SchemaWithMetadata
```

Comment:
I think we don't use it?

Reply:
we use it in ConfluentUniversalKafkaDeserializer

Reply:
true, had it hidden :)